.github/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 1. [GENERAL] Fixed issue in C++ WASM Framework that caused performance degradation in some WASM modules - @frankkopp (Frank Kopp)
 1. [FCU] Fixed auto-initialisation of baro unit - @tracernz (Mike)
+1. [GENERAL] Move Navigraph Auth to shared systems - @MicahBCode (Mischa Binder)
```

Comment:
This doesn't need a changelog as there's no user-visible change.

Reply:
Good point. Is removed now 👍